sys-apps/ipmicfg/ipmicfg-1.27.1.170901.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit versionator
```

Comment:
Please use `eapi7-ver` instead (it's forward-compat for EAPI 7 version functions).

Reply:
Changed to eapi7-ver by newest push.